### ISSUE TITLE
Allow bypass of max gas limit on test nodes

### ISSUE THREAD

**Issue by jtolotti-figure** [provenance-io/provenance#1000]
## Summary

Create a configuration setting that would disable the maxGas restriction on test nodes

## Problem Definition

When running tests using a local version of Provenance, it is not possible to store the wasm for a smart contract because the gas fee exceeds the maximum allowed gas value. 

## Proposal

Create a configuration parameter that would allow the maximum gas restriction to be bypassed on test nodes in order to prevent the need for a governance proposal.

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by arnabmitra** [comment]
i think this ticket is talking about local nodes(make localnet-start etc) @jtolotti-figure can you confirm? 

To clarify test nodes would be pio-testnet where i am not sure we should run a system different from mainnet imo. Also config file on a node would obv have to be across all the nodes there since that would lead to determinism problems.

**Comment by jtolotti-figure** [comment]
Yes, this is referencing local nodes and yes, this was with respect to the 4m/tx limit and not the total gas limit of the block.
